packages/charts/theme/lumo/vaadin-chart-styles.js

```diff
@@ -0,0 +1,79 @@
+import '../vaadin-chart-base-theme.js';
+import '@vaadin/vaadin-lumo-styles/color.js';
+import '@vaadin/vaadin-lumo-styles/typography.js';
+import { css, registerStyles } from '@vaadin/vaadin-themable-mixin/register-styles.js';
+
+registerStyles(
+  'vaadin-chart',
+  css`
+    :host {
```

Comment:
Please conisder exporting the `css` literal alongside with passing it to `registerStyles`, e.g. like this:

```js
const chartColors = css`
  :host {
    /* colors variables */ 
  }
`;

const chartTheme = css`
  :host {
    /* other variables */ 
  }
`;

registerStyles('vaadin-chart', [chartColors, chartTheme], { moduleId: 'lumo-chart' });

export { chartColors, chartTheme }
```

Reply:
Done.